NDTensors/src/arraystorage/arraystorage/tensor/svd.jl

```diff
@@ -51,7 +51,7 @@ function tsvd(
   # Only used by BlockSparse svd
   min_blockdim=nothing,
 )
-  alg = replace_nothing(alg, default_svd_alg(a))
+  alg = replace_nothing(alg, default_svd_alg(a, unwrap_type(a)))
```

Comment:
If you define `default_svd_alg(a) = default_svd_alg(unwrap_type(a), a)` then you can just keep this line as it is.